examples/06_bmg_flash_attention/xe_fmha_fwd_runner.hpp

```diff
@@ -87,6 +88,7 @@ struct Options {
     }
 
     cmd.get_cmd_line_argument("scheduler", scheduler, std::string("Individual"));
+    cmd.get_cmd_line_argument("layout", layout, std::string("NHD"));
```

Comment:
Somewhere we need error-checking to make sure that layout is either "NHD" or "HND".

Reply:
fixed https://github.com/intel/sycl-tla/pull/603/commits/3aab22849aea781e3b33b074b1573a692e2a146f